### PR TITLE
Add docs for new schema endpoints

### PR DESCRIPTION
**Pull request by azrikahar**
Docs for directus/directus#14713.

Closes DOCS-149

## Changes

- [x] add API references for `/schema/snapshot`, `/schema/apply`, and `/schema/diff`
- [x] add mentions of YAML export

### CODE REVIEW

File: configuration/activity-log.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ can also filter items by activity from the Navigation Bar.
 
 To ensure proper accountability, system collections are **read only** by design. However, users with an Admin role have
 the ability to reopen, view, and modify an item's values in activities from non-system collections _(where the name does
-not begin with `directus_`)_. To view or modify an activity, follow these steps.
+not begin with `directus_`)\_. To view or modify an activity, follow these steps.
```
Comment (azrikahar) [inline configuration/activity-log.md:83]:
```suggestion
not begin with `directus_`)_. To view or modify an activity, follow these steps.
```

Removing odd unrelated change.

File: reference/system/server.md [hunk 2]
```diff
@@ -349,3 +349,398 @@ query {
...
+JSON object containing [collections](/reference/system/collections#the-collection-object),
+[fields](/reference/system/fields#the-field-object), and [relations](/reference/system/relations#the-relation-object) to
+apply.
+
+Alternatively, send a JSON or YAML schema file in a `multipart/form-data` request. See
+[Upload a File](/reference/files#upload-a-file) for more information.
+
+### Returns
+
+Empty body.
```
Comment (paescuj) [inline reference/system/server.md:609]: Will return the diff

**Review by rijkvanzanten — APPROVED** [review]
